recipe/build-libadios.sh

```diff
@@ -66,37 +47,38 @@ if [[ "${CONDA_BUILD_CROSS_COMPILATION}" == "1" ]]; then
     export CMAKE_ARGS="${CMAKE_ARGS} -DADIOS2_HAVE_ZFP_CUDA_EXITCODE=0"
 fi
 
-
-cmake              \
-    -S ${SRC_DIR}  \
-    -B build       \
-    ${CMAKE_ARGS}  \
+# shellcheck disable=SC2086
+cmake               \
+    $CMAKE_ARGS     \
+    -S "$SRC_DIR"   \
+    -B build        \
+    -GNinja         \
     -DCMAKE_BUILD_TYPE=Release                \
-    -DCMAKE_VERBOSE_MAKEFILE=ON               \
```

Comment:
Please keep `-DCMAKE_VERBOSE_MAKEFILE=ON` for easier debugging.

Reply:
We are using ninja for the unix builds